libraries/AP_Landing/AP_Landing.cpp

```diff
@@ -445,8 +637,28 @@ void AP_Landing::log(void) const
     case TYPE_STANDARD_GLIDE_SLOPE:
         type_slope_log();
         break;
+    case TYPE_DEEPSTALL:
```

Comment:
Nothing to log at the moment. I was planning on moving all those debug print values into a log at some point. However I think we need to do the rethink of the logging structure first, as this interface for logging is actually a poor fit on deepstall.